pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computefirewallpolicy/_http.log

```diff
@@ -0,0 +1,455 @@
+POST https://www.googleapis.com/compute/v1/locations/global/firewallPolicies?alt=json&parentId=organizations%2F${organizationID}
```

Comment:
I like that you recorded the real GCP log first and committed it - it makes it easy to see the diffs 👍